app.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Hello Kizito, what does the app.json file do?

Reply:
@frostyblok the `app.json` file contains configurations for Heroku, that are needed to create review apps from the repository every time a new PR is raised.

Reply:
@frostyblok, good question.

The `app.json` is required by Heroku for the pipeline to create review apps based off of our staging branch

Reply:
Oh, I didn't know that, thank you! I will be implementing that on my CP